### PR TITLE
Fixes Cross Tenant Validation for Comments and UX when no Comments and SDKs exist

### PR DESCRIPTION
**Pull request by sanjulamadurapperuma**
Comments

image

image

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Sdk.jsx [hunk 1]
```diff
@@ -246,22 +254,28 @@ class Sdk extends React.Component {
...
+                    <Typography variant='h5' component='h3'>
+                        <FormattedMessage
+                            id='Apis.Details.Sdk.no.sdks'
+                            defaultMessage='No SDKs Yet'
+                        />
+                    </Typography>
+                    <Typography component='p'>
+                        <FormattedMessage
+                            id='Apis.Details.Sdk.no.sdks.content'
+                            defaultMessage='No SDKs available for this API yet'
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Sdk.jsx]:
```suggestion
                            defaultMessage='No SDKs available for this API'
```

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Sdk.jsx [hunk 2]
```diff
@@ -246,22 +254,28 @@ class Sdk extends React.Component {
...
-                        </Paper>
-                    </Grid>
-                </Grid>
-            </Paper>
+            <div className={classes.genericMessageWrapper}>
+                <InlineMessage type='info' className={classes.dialogContainer}>
+                    <Typography variant='h5' component='h3'>
+                        <FormattedMessage
+                            id='Apis.Details.Sdk.no.sdks'
+                            defaultMessage='No SDKs Yet'
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Sdk.jsx]:
```suggestion
                            defaultMessage='No SDKs'
```

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/locales/en.json [hunk 3]
```diff
@@ -93,7 +94,8 @@
   "Apis.Details.Overview.subscribe.to.application.btn": "Subscribe to an Application",
   "Apis.Details.Overview.subscriptions": "Subscriptions >>",
   "Apis.Details.Overview.view.credentials": "View Credentials",
-  "Apis.Details.Sdk.no.lanuages": "No languages are configured.",
+  "Apis.Details.Sdk.no.sdks": "No SDKs Yet",
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/locales/en.json]:
```suggestion
  "Apis.Details.Sdk.no.sdks": "No SDKs",
```

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/locales/en.json [hunk 4]
```diff
@@ -93,7 +94,8 @@
...
+  "Apis.Details.Sdk.no.sdks.content": "No SDKs available for this API yet",
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/locales/en.json]:
```suggestion
  "Apis.Details.Sdk.no.sdks.content": "No SDKs available for this API",
```

Comment (sanjulamadurapperuma) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/locales/en.json]: Thanks, the suggested changes have been made and merge conflicts resolved.

**Pull request merged**